src/jsonGraphqlExpress.js

```diff
@@ -1,8 +1,5 @@
 import graphqlHTTP from 'express-graphql';
-import { printSchema } from 'graphql';
-import { makeExecutableSchema } from 'graphql-tools';
-import getSchemaFromData from './introspection/getSchemaFromData';
-import resolver from './resolver';
+import schemaBuilder from './schemaBuilder';
```

Comment:
Not very important but I would rename that `buildSchema` as we don't need to instantiate a builder